formulas/wick-base/functions/wick-make-user

```diff
@@ -21,6 +21,10 @@
 # If the given user already exists and no options are given `usermod` will not
 # be run.
 #
+# When `--groups` is specified the list of groups specified is added to the
```

Comment:
Could you changed a "specified" word to something else? Perhaps change the first to "used" or "included"? Or the second could be "indicated" or removed entirely?